examples/serial/shallow_water/galewsky.py

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
might this file be better called galewsky_serial or something?

Reply:
Yes, it would be.